docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -328,7 +460,51 @@ Or to set the column ``my_new_partition_column`` as a partition column on a tabl
 
 The current values of a table's properties can be shown using :doc:`SHOW CREATE TABLE </sql/show-create-table>`.
 
-.. _iceberg-type-mapping:
+COMMENT
+~~~~~~~
+
+The Iceberg connector supports setting comments on the following objects:
+
+- tables
+- views
+- columns
+
+The connector uses the command :doc:`COMMENT </sql/comment>` for setting
+comments.
+
+Setting comments on tables or columns is not supported in case that the
+Iceberg connector is using the :ref:`Glue catalog <iceberg-glue-catalog>`.
+Setting comments on view columns is not supported.
+
+.. _iceberg-data-management:
+
+Data management
+^^^^^^^^^^^^^^^
+
+The :ref:`sql-data-management` functionality includes support for ``INSERT``,
+``UPDATE``, ``DELETE``, and ``MERGE`` statements.
+
+.. _iceberg-delete:
+
+Deletion by partition
+~~~~~~~~~~~~~~~~~~~~~
+
+For partitioned tables, the Iceberg connector supports the deletion of entire
+partitions if the ``WHERE`` clause specifies filters only on the identity-transformed
+partitioning columns, that can match entire partitions. Given the table definition
+above, this SQL will delete all partitions for which ``country`` is ``US``::
+
+    DELETE FROM iceberg.testdb.customer_orders
+    WHERE country = 'US'
+
+Tables using either v1 or v2 of the Iceberg specification will perform a partition
```

Comment:
Drop the mention of the spec since it works for both anyway.

Also change to present tense